sql/core/src/test/gen-java/org/apache/spark/sql/execution/datasources/parquet/test/avro/AvroArrayOfArray.java

```diff
@@ -129,6 +129,7 @@ public boolean hasIntArraysColumn() {
     }
 
     @Override
+    @SuppressWarnings(value="unchecked")
```

Comment:
I see, so this would be overwritten if we regenerated the sources. Weird because it generates `@SuppressWarnings("all")` at the top. I agree, stay consistent with what's generated then, I guess.